go/tasks/plugins/hive/executor_metrics.go

```diff
@@ -21,11 +22,13 @@ var (
 func getQuboleHiveExecutorMetrics(scope promutils.Scope) QuboleHiveExecutorMetrics {
 	return QuboleHiveExecutorMetrics{
 		Scope: scope,
-		ReleaseResourceFailed: labeled.NewCounter("released_resource_failed",
+		ResourceReleased: labeled.NewCounter("resource_release_success",
+			"Resource allocation token released", scope),
```

Comment:
```suggestion
			"Resource allocation token released", scope, labeled.EmitUnlabeledMetric),
```

and same for the other ones